src/components/Menu.tsx

```diff
@@ -132,7 +132,7 @@ const Menu: React.FC<MenuProps> = ({
             }}
           >
             <IonIcon slot="start" icon={hammer} />
-            Show Tutorial
+            <span className="tutorial-label">Show Tutorial</span>
```

Comment:
The Angular app uses an `ion-label`. Should we use that here instead for consistency?

https://github.com/ionic-team/ionic-conference-app/blob/d74e556278264470856891c2f072186161aa5198/src/app/app.component.html#L102